app/core/tests/gae_suite.py

```diff
@@ -35,7 +35,7 @@
 OPPIA_TOOLS_DIR = os.path.join(CURR_DIR, '..', 'oppia_tools')
 
 DIRS_TO_ADD_TO_SYS_PATH = [
-    CURR_DIR,
+    CORE_DIR,
```

Comment:
I totally forgot about the reason why I made this change.
Since we move most Python files into `/app`, we need to add the path to `/app` to  gae_suite's sys.path so that we can run individual test file/classes.

Reply:
Can you talk through the reasoning behind that a bit more? I think maybe it's actually due to a missing `__init__.py` file within the app/ folder?

I'm a bit concerned about changing the syspath here because the imports in the files seem to be based on the rootdir ("from core.X.Y import Z" rather than "from X.Y import Z") and it should probably be consistent throughout.

Reply:
The command I am trying to run is
`bash scripts/run_backend_tests.sh --test_target='core.controllers.base_test.BaseHandlerTest'`

I did try to copy `__init__.py` first through in `/app` folder and it did not help.
The working solution was adding CORE_DIR.
Currently, the folder tree looks like this
/foundation-website
- /app
-- /core
---- /controllers
---- /domain
....
So I think it just affects running gae_suite?